### DISCUSSION TITLE
Suggestions don't work as intended, please help

### DISCUSSION THREAD

**Discussion by 20170010** [florisboard/florisboard#2148] [Q&A]
#### Short description
Suggestions aren't shown, no matter what settings i try to use, it just doesn't work

#### Steps to reproduce
Nothing really to do, tried to show suggestions, doesn' work

#### Environment information
- FlorisBoard Version: newest and tried it as well with 0.4.0 alpha 3
- Install Source: Github .zip
- Device: Google Pixel 5 API 30
- Android: 11

#### Picture
![image](https://user-images.githubusercontent.com/112947872/215103505-229204c1-9780-495c-b9d6-c50afac4ce3e.png)

I tried to change the suggestion method in the LatinLanguageProvider class, because I oticed that that is were the suggestions were built and at first I thought I broke something but I got myself a clean new version from the GitHub and it's still not working as intended.

**Comment by tsiflimagas** [comment]
Word suggestions aren't implemented yet, that's why you don't get any. They will come at some point.

**Reply by 20170010** [reply]
But they did work at some point and suddenly they weren't working anymore?

**Reply by 20170010** [reply]
The "next0", "next1" stuff was already working and I don't know why it suddenly stopped working

**Comment by Beach77** [comment]
> Word suggestions aren't implemented yet, that's why you don't get any. They will come at some point. 

That has been said for like a year now and I have honestly lost all hope of it ever happening

**Reply by MuteMuteh** [reply]
The project is dead at this point, they promised monthly releases for their alpha channel but they haven't updated it since last year.

**Reply by BHydden** [reply]
If you check their matrix it's still active dev has just been busy and this current step has been complicated. Seems confident that something will be out Soon™ though.
